.ls-lint.yml

```
@@ -0,0 +1,13 @@
+ls:
+  .js: regex:[+_]?[a-z0-9\-\.]+
+  .ts: regex:[+_]?[a-z0-9\-\.]+
+  .d.ts: kebab-case
+  .svelte: regex:[+_]?[a-z0-9\-\.]+
+
+ignore:
+  - .git
+  - .histoire
+  - .svelte-kit
+  - .vercel
+  - build
+  - node_modules
```

Wondering if we want to add the following 
```suggestion
  - node_modules
  - dist
  - server
```

And maybe even `playwright-report` for when running the command locally.